x/ccv/provider/keeper/hooks_test.go

```diff
@@ -53,14 +53,14 @@ func TestValidatorConsensusKeyInUse(t *testing.T) {
 
 		gomock.InOrder(
 			mocks.MockStakingKeeper.EXPECT().GetValidator(ctx,
-				newValidator.SDKStakingOperator(),
+				newValidator.SDKStakingValidator().GetOperator(),
 			).Return(newValidator.SDKStakingValidator(), true),
 		)
 
 		tt.setup(ctx, k)
 
 		t.Run(tt.name, func(t *testing.T) {
-			if actual := providerkeeper.ValidatorConsensusKeyInUse(&k, ctx, newValidator.SDKStakingOperator()); actual != tt.expect {
+			if actual := providerkeeper.ValidatorConsensusKeyInUse(&k, ctx, newValidator.SDKStakingValidator().GetOperator()); actual != tt.expect {
```

Comment:
Is it intentional to pass in an operator address to a method that seems like it accepts a consensus key? 

If this is intentional, we should add a comment or change the naming of that method

Reply:
It's intentional. We could rename this function to sthg like `IsConsensusAddressOfValidatorInUse`. Wdyt?

Reply:
Your idea works, also something like `IsValAddrInUse` could work, since the addrs we're iterating over shouldn't matter to the caller. Changing the parameter name from `valAddr` to `valOpAddr` [here](https://github.com/cosmos/interchain-security/blob/a5b78e375e089773d65f75636e9c36dc0b3a1b9b/x/ccv/provider/keeper/hooks.go#L70) is useful too

Aside: It seems like the `sdk.ValAddr` type should always imply operator address? I didn't know this previously. Cons addresses could still be assigned to that type unfortunately